docs/concepts/overview.md

```diff
@@ -88,31 +88,6 @@ operator facing, but may optionally offer an application developer facing
 composite resource claim that acts as a proxy for a composite resource. You can
```

Comment:
There are mentions of OAM and its CRDs in the introduction section. Would you like to remove them as well?

Reply:
@muvaf no I don't think so because those notes are alluding the the fact that helm will install OAM CRDs even if the subchart is not enabled